### DISCUSSION TITLE
Suggestion: Collision Exclusion

### DISCUSSION THREAD

**Discussion by Msplats** [replit/kaboom#659] [Ideas]
Is there currently a way to make only specific items interact with each other? 
(I can't seem to find anything like it in the documentation)
For example, A 2d sidescroller, where all the players can jump from platform to platform but can pass through each other.

If there isn't a way to already do this, I think it would be a cool addition.

**Comment by lajbel** [comment]
Yeah, it's already here a function for that in Kaboom 3000 (not released yet)

```js
area({
    collisionIgnore: ["platform"]
})
```
